src/shared/@types/store.ts

```diff
@@ -65,11 +65,17 @@ export type TNetworkProfile = {
   name: string
   networkByBlockchain: TSelectedNetworks
 }
+
+export type TOverTheAirInfo = {
+  lastAppVersion?: string
+  hasUpdated?: boolean
```

Comment:
Why can it be undefined? Should be better using only the `true` or `false` I think.